docs/diff_output.md

```diff
@@ -0,0 +1,84 @@
+| added | 0.0.0.0-255.255.255.255 | default/backend[Deployment] | No Connections | TCP 9090 |  |
+```
+
+```
+$ ./bin/k8snetpolicy diff --dir1 tests/netpol-analysis-example-minimal/ --dir2 tests/netpol-diff-example-minimal/ -o csv
+
+diff-type,source,destination,dir1,dir2,workloads-diff-info
+changed,default/frontend[Deployment],default/backend[Deployment],TCP 9090,"TCP 9090,UDP 53",
+added,0.0.0.0-255.255.255.255,default/backend[Deployment],No Connections,TCP 9090,
+```
+
+```
+$ ./bin/k8snetpolicy diff --dir1 tests/netpol-analysis-example-minimal/ --dir2 tests/netpol-diff-example-minimal/ -o dot
+
+digraph {
+        "0.0.0.0-255.255.255.255" [label="0.0.0.0-255.255.255.255" color="blue" fontcolor="blue"]
+        "default/backend[Deployment]" [label="default/backend[Deployment]" color="blue" fontcolor="blue"]
+        "default/frontend[Deployment]" [label="default/frontend[Deployment]" color="blue" fontcolor="blue"]
+        "0.0.0.0-255.255.255.255" -> "default/backend[Deployment]" [label="TCP 9090" color="#008000" fontcolor="#008000"]
+        "0.0.0.0-255.255.255.255" -> "default/frontend[Deployment]" [label="TCP 8080" color="grey" fontcolor="grey"]
+        "default/frontend[Deployment]" -> "0.0.0.0-255.255.255.255" [label="UDP 53" color="grey" fontcolor="grey"]
+        "default/frontend[Deployment]" -> "default/backend[Deployment]" [label="TCP 9090,UDP 53 (old: TCP 9090)" color="magenta" fontcolor="magenta"]
+        nodesep=0.5
+        subgraph cluster_legend {
+                label="Legend"
+                fontsize = 10
+                margin=0
+                a [style=invis height=0 width=0]
+                b [style=invis height=0 width=0]
+                c [style=invis height=0 width=0]
+                d [style=invis height=0 width=0]
+                e [style=invis height=0 width=0]
+                f [style=invis height=0 width=0]
+                g [style=invis height=0 width=0]
+                h [style=invis height=0 width=0]
+                {rank=source a b c d}
+                {rank=same e f g h}
+                a -> b [label="added connection", color="#008000" fontcolor="#008000" fontsize = 10 arrowsize=0.2]
+                c -> d [label="removed connection", color="red" fontcolor="red" fontsize = 10 arrowsize=0.2]
+                e -> f [label="changed connection", color="magenta" fontcolor="magenta" fontsize = 10 arrowsize=0.2]
+                g -> h [label="non-changed connection", color="grey" fontcolor="grey" fontsize = 10 arrowsize=0.2]
+                np [label="new peer" color="#008000" fontcolor="#008000" fontsize = 10]
+                lp [label="lost peer" color="red" fontcolor="red" fontsize = 10]
+                pp [label="persistent peer" color="blue" fontcolor="blue" fontsize = 10]
+                {rank=sink np lp pp}
+                np->lp [style=invis]
+                lp->pp [style=invis]
+        }

```

Comment:
I would avoid adding this static part of the `dot` output for the example output

Reply:
please create also an `svg` file from the `dot` file, add here the explanation that from `dot` using graphviz the graph can be produced, and then add as example the command to generate the `svg` file, and also add the `svg` graph here as well. (so that the final graph can be viewed in the readme itself).